MonoGame.Framework/Android/MonoGameAndroidGameView.cs

```diff
@@ -656,14 +656,7 @@ void UpdateAndRenderFrame()
                 updateEventArgs.Time = t < 0 ? 0 : t;
             }
 
-            try
-            {
-                UpdateFrameInternal(updateEventArgs);
-            }
-            catch (Content.ContentLoadException)
-            {
-                // ignore it..
-            }
+            UpdateFrameInternal(updateEventArgs);
```

Comment:
This is handled on https://github.com/MonoGame/MonoGame/pull/5974

Reply:
If you just let the exception raise and we are rendering on a separate thread, the app will not stop. So we really need to throw this on the UI thread (if needed)